src/test/java/de/qabel/desktop/ui/actionlog/ActionlogControllerTest.java

```diff
@@ -1,25 +1,117 @@
 package de.qabel.desktop.ui.actionlog;
 
 import de.qabel.core.config.Account;
+import de.qabel.core.config.Contact;
 import de.qabel.core.config.Identity;
-import de.qabel.core.crypto.QblECKeyPair;
+import de.qabel.core.drop.DropMessage;
+import de.qabel.core.drop.DropURL;
+import de.qabel.core.exceptions.*;
 import de.qabel.desktop.ui.AbstractControllerTest;
+import javafx.scene.layout.VBox;
+import org.junit.Before;
 import org.junit.Test;
 
-import static org.junit.Assert.assertNotNull;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Date;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
 
 
 public class ActionlogControllerTest extends AbstractControllerTest {
 
 	ActionlogController controller;
+	Identity i;
+	ActionlogView view;
+	Contact c;
+	String fakeURL;
+	String workingURL;
+	String text = "MessageString";
 
-	@Test
-	public void injectTest() {
-		ActionlogView view = new ActionlogView();
-		Identity i = new Identity("test", null, new QblECKeyPair());
+	@Before
+	public void setUp() throws Exception {
+		super.setUp();
+		fakeURL = "http://localhost:12345/abcdefghijklmnopqrstuvwxyzabcdefgworkingUrl";
+		workingURL = "https://qdrop.prae.me/abcdefghijklmnopqrstuvwxyzabcdefgworkingUrl";
```

Comment:
Please don't use the production server for tests.
With https://github.com/Qabel/qabel-desktop/commit/f87c0d73a254cdc167f8d5192db4e775dc7c9be7 the local server should be usable again.